.github/workflows/pythonbuild.yml

```diff
@@ -261,6 +261,9 @@ jobs:
           FLYTEKIT_IMAGE: localhost:30000/flytekit:dev
           FLYTEKIT_CI: 1
           PYTEST_OPTS: -n2
+          AWS_ENDPOINT_URL: 'http://localhost:30002'
+          AWS_ACCESS_KEY_ID: minio
+          AWS_SECRET_ACCESS_KEY: miniostorage
         run: |
```

Comment:
Could you also add the plugin to this [list](https://github.com/flyteorg/flytekit/blob/23beed9d9e484eaa88c24a9d01a6638a3e7525b5/.github/workflows/pythonbuild.yml#L310)